### DISCUSSION TITLE
Error: ENOENT: no such file or directory

### DISCUSSION THREAD

**Discussion by YuZhang-steven** [pmndrs/zustand#2643] [Bug report]
### Bug Description

I use zustand in a Next.js project and get the following error after import the zustand.
![image](https://github.com/user-attachments/assets/2eadcc9e-b780-4b93-8984-8548fbe7ebed)

### Reproduction Link

/

**Comment by dbritto-dev** [comment]
@YuZhang-steven as far I see that error is coming from `tunnel-rat` per deeps

**Reply by YuZhang-steven** [reply]
Hi @dbritto-dev ! Thanks for replying quickly! Do you know why tunnel-rat looking for Zustand in its own folder? I am a little bit confused what's my next step to address this problem? Do you have any suggestions?

**Reply by dbritto-dev** [reply]
@YuZhang-steven do you have zustand installed?

**Reply by YuZhang-steven** [reply]
Yep
